### PR TITLE
release: 4.1.1

### PR DESCRIPTION
**Pull request by theopolis**
~~TODO: The windows hash is invalid.~~

Ready for review!

### CODE REVIEW

File: src/data/osquery_package_versions/4.1.1.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "version": "4.1.1",
+  "downloads": {
```
Comment (directionless) [inline src/data/osquery_package_versions/4.1.1.json]:
```suggestion
  "url": "https://pkg.osquery.io",
  "downloads": {
```

File: src/data/osquery_metadata.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "current_version":"4.0.2",
+  "current_version":"4.1.1",
```
Comment (directionless) [inline src/data/osquery_metadata.json]: I think if we commit without this, we'll be able to see the website workflow

**Comment by directionless** [comment]
hashes look correct. But I think it's needs the URL key before merge

File: src/data/osquery_metadata.json (hunk 2 above)
Comment (theopolis) [reply src/data/osquery_metadata.json]: I'm not sure what you mean.

Comment (directionless) [reply src/data/osquery_metadata.json]: For future reference, I think we can update the website to include the new packages, but not change what the current release points to

**Pull request merged**